code/index.html

```diff
@@ -1,30 +1,120 @@
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Project 3</title>
+		<link rel="stylesheet" href="style.css" />
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+		<meta name="viewport" content="width=device-width, initial-scale=1">  
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Kaisei+HarunoUmi&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Noto+Serif&display=swap" rel="stylesheet">
+	</head>
+	
+	<form action="https://httpbin.org/anything" method="post"></form>
+	
+	<body>
+		<div class="hero-image">
+			<div class="hero-text">
+				<h1>Your Dog's Closet</h1>
+				<p>Let's double your dog's fabulousness</p>
+			</div>
+		</div>
+
+		<div class="navbar">
+			<li class="list"> <a class="link" href="#home">Home</a></li>
+			<li class="list"><a class="link" href="#news">News</a></li>
+			<li class="list"><a class="link" href="#contact">Contact</a></li>
+			<li class="list"><a class="link" href="#about">About</a></li>
+		  </div>
+
+		  <form action="https://httpbin.org/anything" method="POST">
+
+			
+<div class="form-wrapper">
+<div class= "sign-up-form">
+	<div class="sign-up-header">
+	  <h1>Sign Up</h1>
+	</div>
+	<div class="sign-up-body">
+	  <label for="name"><b>Name</b></label>
+      <input class="placeholder" type="text" placeholder="First and Last name" name="name" required>
+
+      <label for="email"><b>Email</b></label>
+      <input class="placeholder" type="text" placeholder="Enter Email" name="email" required>
+
+      <label for="psw"><b>Password</b></label>
+      <input class="placeholder" type="password" placeholder="Enter Password (8 characters minimum)" minlength="8" name="psw" required>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <label for="psw-repeat"><b>Repeat Password</b></label>
+      <input class="placeholder" type="password" placeholder="Repeat Password" minlength="8" name="psw-repeat" required>
+      
+	  <div class= radio>
+		  <div>
+	  <label for="radio"><b>Male</b></label>
+	  <input type="radio" id="gender" name="gender" value="male" checked> 
+	    </div>
+		<div>
+	  <label for="radio"><b>Female</b></label>
+	  <input type="radio" id="gender" name="gender" value="female">
+	</div>
```

Comment:
Great job on your form, you have used several different input types and I get a message if I've missed something.